lib/sumomo.rb

```diff
@@ -250,5 +260,6 @@ def self.get_stack_outputs(name:, region:)
     map
   end
 
-  singleton_class.send(:alias_method, :create_stack, :update_stack)
+  singleton_class.send(:alias_method, :create_stack, :create_stack)
+  singleton_class.send(:alias_method, :update_stack, :update_stack)
```

Comment:
This isn't necessary anymore. We can delete the aliases.

Reply:
@davidsiaw 
I see.

I have removed them.